roles/dev_datastore/datastore_setup/vars/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+packages:
+  - git
+  - redis-server
+  - libpq-dev
+  - libxml2-dev
+  - libxslt-dev
+  - libevent-dev
+
+github_repo: https://github.com/IATI/IATI-Datastore.git
+
+user_home_dir: /home/datastore/
```

Comment:
The username part of this looks like something that should be able to come from a variable containing the name of the user.